docs/tutorials/run-kinto.rst

```diff
@@ -0,0 +1,107 @@
+.. _run-kinto:
+
+Run Kinto
+#########
+
+Mozilla demo server
+===================
+
+A Kinto instance is running at https://kinto.dev.mozaws.net/v1/
+
+It should be enough to get started, but the records are flushed every night.
+
+
+Using Docker
+============
+
+If you have Docker, Kinto can be started locally with a single command:
+
+::
+
+    sudo docker run -p 8888:8888 kinto/kinto-server
+
+The server should now be running on http://localhost:8888
+
+It is possible to specify every Kinto setting through environment variables.
+For example, using an environment file:
+
+::
+
+    # kinto.env
+    KINTO_USERID_HMAC_SECRET = tr0ub4d@ur
+    KINTO_BATCH_MAX_REQUESTS = 200
+
+And running the container with ``docker run --env-file ./kinto.env ...``
+
+
+Using Python package
+====================
+
+Python tooling
+--------------
+
+The following tools are necessary to initiate the local installation and use
+our helpers:
+
+* `python-virtualenv`
```

Comment:
Nit: We may want to describe further how to actually install it, or at least link to external docs providing this information.